ui/apps/dashboard/src/components/Insights/InsightsStateMachineContext/useFetchInsights.ts

```diff
@@ -16,8 +16,8 @@ export interface FetchInsightsParams {
 
 type FetchInsightsCallback = (query: string, name: undefined | string) => void;
 
-const insightsQuery = graphql(`
-  query Insights($query: String!, $workspaceID: ID!) {
```

Comment:
I think this was causing the conflict with https://github.com/inngest/monorepo/pull/5257, but it's best to avoid by not having types end in `Query` anyway